src/app/pages/datasets/components/dataset-form/dataset-form.component.html

```diff
@@ -4,7 +4,7 @@
   [loading]="isLoading"
 ></ix-modal-header>
 
-<mat-card>
+<mat-card *ngIf="!guiService.hasError">
```

Comment:
It would be nice to keep templates clean and handle error cases using directive.